Mathlib/Algebra/Order/CompleteField.lean

```diff
@@ -155,7 +155,8 @@ theorem cutMap_add (a b : α) : cutMap β (a + b) = cutMap β a + cutMap β b :=
       rw [coe_mem_cutMap_iff]
       exact_mod_cast sub_lt_comm.mp hq₁q
   · rintro _ ⟨_, _, ⟨qa, ha, rfl⟩, ⟨qb, hb, rfl⟩, rfl⟩
-    refine' ⟨qa + qb, _, by norm_cast⟩
+    -- After leanprover/lean4#2734, `norm_cast` needs help with beta reduction.
```

Comment:
This looks like a possibly regression to me (and the only one in this patch); can you explain what's going on in the PR description, and maybe open a tracking issue for fixing this to be referenced there too?

Reply:
Tracking issue at #8052.

(Also, since you read these diffs I've changed these regressions to use `beta_reduce` or `unfold_let` specifically, instead of a blind `dsimp`.)